.python-version

```diff
@@ -0,0 +1 @@
+3.4.9
```

Comment:
Does pyenv allow you to comment this file? It's not immediately obvious that this is for pyenv, how it's supposed to work, or when we should update this.

Reply:
I'll look into that. At least you can find out via git-blame :-P

Reply:
pyenv doesn't like it when you add a comment to `.python-version`, so I just updated the code style guidelines with an explanation in #15173.